ui/app/src/components/GroupsManagement/GroupsManagement.tsx

```diff
@@ -103,7 +103,7 @@ export default function GroupsManagement() {
   return (
     <section>
       <GlobalAppToolbar
-        title={<FormattedMessage id="GlobalMenu.Groups" defaultMessage="Groups" />}
+        title={<FormattedMessage id="globalMenu.groups" defaultMessage="Groups" />}
```

Comment:
Same as comment as above

Reply:
in i18n-legacy.ts we have `home.globalMenu.groups`, I thought we were using GlobalMenu.Groups. That is why I changed all of them, we can use the same as i18n-legacy.ts or keep it using globalMenu.*